asv2/README.md

```diff
@@ -35,7 +35,7 @@ The following steps use the Celo [ContractKit](https://docs.celo.org/developer/c
     npm install @celo/identity
     ```
 
-2. Set up your issuer, which is the account registering attestations. When a user requests for the issuer to register an attestation, the issuer should [verify](protocol.md#verification) somehow that the user owns their identifier (ex. SMS verification for phone number identifiers).
+2. Set up your issuer (read "Authentication" section in [privacy.md](privacy.md#authentication)), which is the account registering attestations. When a user requests for the issuer to register an attestation, the issuer should [verify](protocol.md#verification) somehow that the user owns their identifier (ex. SMS verification for phone number identifiers).
```

Comment:
I think this should be linked in the next step (step 3) instead, as interactions with ODIS and using the auth signer isn't actually done here

Reply:
I can't seem to add the suggestion directly in this PR, but I would add to step 3:

> See the privacy doc for details on [authentication](privacy.md#authentication) and the [service context](privacy.md#service-context)